arm-customer-insights/2016-01-01/swagger/customer-insights.json

```diff
@@ -0,0 +1,4735 @@
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/widgetTypes/{widgetTypeName}": {
+      "get": {
+        "tags": [
+          "WidgetTypes"
+        ],
+        "operationId": "WidgetTypes_Get",
+        "x-ms-examples": { 
+          "WidgetTypes_Get": { "$ref": "../examples/WidgetTypesGet.json" } 
+        },
+        "description": "Gets a widget type in the specified hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "name": "widgetTypeName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the widget type."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/WidgetTypeResourceFormat"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.CustomerInsights/hubs/{hubName}/views": {
+      "get": {
+        "tags": [
+          "Views"
+        ],
+        "operationId": "Views_ListByHub",
+        "x-ms-examples": { 
+          "Views_ListByHub": { "$ref": "../examples/ViewsListByHub.json" } 
+        },
+        "description": "Gets all available views for given user in the specified hub.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "hubName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the hub."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "userId",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The user ID. Use * to retreive hub level views."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "sc
```

Comment:
If in swagger file we keep the cases for field names same as them in RP response, we may be able to enable x-ms-client-flatten now, and won't introduce breaking change to SDK users if later we change the cases in our RP. Do you think it's an acceptable approach?